src/rard/static/js/project.js

```diff
@@ -345,6 +346,12 @@ function initRichTextEditor($item) {
     let html = $item.find(".ql-editor").html();
     $for.text(html);
   });
+  // When using htmx we can't hook into on submit,
```

Comment:
If you mean when the form submits you'd like htmx to do something, it is possible; I had to do this with the inline form to update definite/possible and book assignment. It's in src/rard/templates/research/partials/render_inline_worklink_form.html if you'd like to look. I might be misunderstanding the usage here

Reply:
Hmm, not sure if this is the same. Prior to using htmx, the js had hooked into on submit to take the editor content and update the introduction field that was actually getting submitted with the form. Htmx doesn't trigger on submit, but instead triggers the htmx:configRequest event which allows you to edit the form fields before they're sent.